### DISCUSSION TITLE
Please let us op-out of the feed

### DISCUSSION THREAD

**Discussion by beeman** [community/community#37590] [Other Feature Feedback, Questions, & Ideas]
The GitHub timelines used to be one of my favorites, the feed is a really bad replacement. I don't care about any of the people you suggest me to follow. Please don't try to become like Twitter.

**Comment by Naville** [comment]
+1, ooof

**Comment by tylearymf** [comment]
+1, ooof

**Comment by dohooo** [comment]
+1

**Comment by devarshishimpi** [comment]
+1

**Comment by martinlindhe** [comment]
yea what gives
